qiita_ware/ebi.py

```diff
@@ -105,21 +86,44 @@ class EBISubmission(object):
     Parameters
     ----------
     preprocessed_data_id : str
+
+    Raises
+    ------
+    EBISumbissionError
+        - If the action is not in EBISubmission.valid_ebi_actions
```

Comment:
I _think_ you don't need to make a list in here to create a list (i.e. remove the `-`)